src/cryptoadvance/specter/templates/base.jinja

```diff
@@ -172,6 +172,7 @@
 			{% endif %}
 		</script>
 	{% endif %}
+	<!-- This scripts block will be overwritten in /device -->
```

Comment:
I'm not sure i understand the comment. The blocks are there to be overwritten. Sure, some of them indeed are. But why do we comment that here?

Reply:
That was a reminder for me. I can remove it.